docs/01-app/01-getting-started/07-fetching-data.mdx

```diff
@@ -504,7 +504,7 @@ export default async function Page({ params }) {
 }
 ```
 
-You can initiate requests in parallel by defining them outside the components that use the data, and resolving them together, for example, with [`Promise.all`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all):
+You can initiate requests concurrently by defining them outside the components that use the data, and resolving them together, for example, with [`Promise.all`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all) (or any of the 4 [`Promise` concurrency methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise#promise_concurrency), depends on your use case):
```

Comment:
Here we can state why these are parallel rather. It is not Promise.all that's the powerful abstraction, but calling fetch('/a') and fetch('/b') right after the other.

```suggestion
Initiate multiple requests by calling fetch and await them together with [`Promise.all`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all). Requests start immediately when fetch is called. `Promise.all` only waits for all of them to resolve, if any rejects `Promise.all` rejects too. If some failures are acceptable, use [`Promise.allSettled`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled) to handle successes and errors together.
```